public/app/features/alerting/unified/hooks/ruleGroup/useUpsertRuleFromRuleGroup.ts

```diff
@@ -38,9 +36,6 @@ export function useAddRuleToRuleGroup() {
       notificationOptions: { successMessage },
     }).unwrap();
 
-    // @TODO remove
-    await dispatch(fetchRulerRulesAction({ rulesSourceName: dataSourceName }));
-
```

Comment:
I need to tests this more but as we redirect to the details page instead of the list page this seems to be unnecessary